src/api/entities/types.ts

```diff
@@ -89,6 +89,11 @@ export interface PaginationOptions {
   start?: string;
 }
 
+export interface MiddlewarePaginationOptions {
```

Comment:
I added this because the start value for middleware needs to be a number while for chain storage it needs to be a string. There are a lot of other places where PaginationOptions is already used, so I didn't alter the current one and instead added new one with that support

Reply:
That makes sense. I missed the different types  😅